lib/manageiq/appliance_console/cli.rb

```diff
@@ -216,14 +216,15 @@ def set_internal_db
       # verify, set up the database with region. activate does it all!
       unless config.activate
         say "Failed to configure internal database"
-        return
+        exit(1)
```

Comment:
Why don't you try raising an exception here and rescuing in the run loop ( line ~179)

That way tests can check for an exception raised

Reply:
Good point @kbrock! Will do.